src/tribler/gui/widgets/downloadspage.py

```diff
@@ -549,6 +551,28 @@ def on_export_download_request_done(self, filename, data):
                 tr("Torrent file exported"), tr("Torrent file exported to %s") % str(dest_path)
             )
 
+    def show_all_headers(self):
```

Comment:
Currently, when I click on "Restore all", I get the following error:
```python
  File "C:\dev\tribler\src\run_tribler.py", line 126, in <module>
    main()
  File "C:\dev\tribler\src\run_tribler.py", line 122, in main
    run_gui(api_port, api_key, root_state_dir, parsed_args)
  File "C:\dev\tribler\src\tribler\gui\start_gui.py", line 89, in run_gui
    exit_code = app.exec_()
  File "C:\dev\tribler\src\tribler\gui\utilities.py", line 463, in trackback_wrapper
    callback(*args, **kwargs)
  File "C:\dev\tribler\src\tribler\gui\widgets\downloadspage.py", line 571, in on_header_right_click_item
    connect(restore_action.triggered, self.show_all_headers)


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\dev\tribler\src\tribler\gui\utilities.py", line 466, in trackback_wrapper
    raise exc from CreationTraceback(traceback_str)
  File "C:\dev\tribler\src\tribler\gui\utilities.py", line 463, in trackback_wrapper
    callback(*args, **kwargs)
TypeError: show_all_headers() takes 1 positional argument but 2 were given
```

To fix it, you need to specify the second unused argument for the signal handler:
```suggestion
    def show_all_headers(self, _=None):
```